CMakeLists.txt

```diff
@@ -41,6 +41,19 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(pybind11-src)
 
+IF (MSVC)
+IF (CMAKE_BUILD_TYPE STREQUAL "RELEASE")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /O2 /Oy")
+set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /O2 /Oy")
+ELSE()
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Ob1")
+set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /Ob1")
+ENDIF()
+
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Zi")
+set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /Zi")
```

Comment:
I'm not a cmake expert, but my understanding is that it doesn't abstract most compiler flags, such as these